boards/arm/nucleo_l452re/doc/index.rst

```diff
@@ -150,10 +150,15 @@ Default Zephyr Peripheral Mapping:
 - UART_1_TX : PA9
 - UART_1_RX : PA10
 - UART_2_TX : PA2
-- UART_2_RX : PA15
+- UART_2_RX : PA3
```

Comment:
This should also be updated in the `pinmux.c`.

Reply:
It's already correct in `pinmux.c`, I guess it was just a typo in the docs!